public/script.js

```diff
@@ -5974,7 +6000,40 @@ export function cleanUpMessage(getMessage, isImpersonate, isContinue, displayInc
     return getMessage;
 }
 
-export async function saveReply(type, getMessage, fromStreaming, title, swipes, reasoning) {
+/**
+ * Adds an image to the message.
+ * @returns {Promise<void>}
+ */
+async function processImageAttachment(message, { parsedImage, imageUrl }) {
+    if (parsedImage?.image) {
+        return saveImageToMessage(parsedImage, message);
```

Comment:
Not necessary, it would be better for readability.
```suggestion
        saveImageToMessage(parsedImage, message);
        return;
```

Reply:
Done.